src/containers/Review/ReviewPage.tsx

```diff
@@ -134,14 +131,11 @@ const ReviewPage: React.FC<{
   const isSubmitted =
     thisReview?.current.reviewStatus === ReviewStatus.Submitted ||
     thisReview?.current.reviewStatus === ReviewStatus.ChangesRequested
-  const isLocked =
-    thisReview?.current.reviewStatus == ReviewStatus.Locked ||
-    thisReview?.current.reviewStatus === ReviewStatus.Discontinued
+  const isLocked = thisReview?.isLocked
```

Comment:
I have a small concern about this change. If this wouldn't show the Review as locked for submission as it should

Reply:
Yes, I mentioned that at the time that we'd need to replace this with some additional check instead. Was in a hurry though and wasn't related to the issue I was fixing so left it to do here. :)

Reply:
Having a quick look through, I can't see anywhere (in both back and front end) where "Discontinued" is ever set. It's referred to a few times, but never written as far as I can tell. 

Should it be?

Reply:
Oh! Let me check this... It should be Discontinued in action `updateReviewStatuses` surely... Is that one of the actions that are gone? So in that case it would need to be added to one back-end trigger...